azurerm/resource_arm_mssql_server_blob_extended_auditing_policies.go

```diff
@@ -0,0 +1,240 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2017-03-01-preview/sql"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	uuid "github.com/satori/go.uuid"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmMsSqlServerBlobExtendedAuditingPolicies() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmMsSqlServerBlobExtendedAuditingPoliciesCreateUpdate,
+		Read:   resourceArmMsSqlServerBlobExtendedAuditingPoliciesRead,
+		Update: resourceArmMsSqlServerBlobExtendedAuditingPoliciesCreateUpdate,
+		Delete: resourceArmMsSqlServerBlobExtendedAuditingPoliciesDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Read:   schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(60 * time.Minute),
+			Delete: schema.DefaultTimeout(60 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"resource_group_name": azure.SchemaResourceGroupName(),
+
+			"server_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: azure.ValidateMsSqlServerName,
+			},
+
+			"state": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice([]string{"Enabled", "Disabled"}, false),
+			},
+
+			"storage_endpoint": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"storage_account_access_key": {
+				Type:      schema.TypeString,
+				Required:  true,
+				Sensitive: true,
+			},
+			"retention_days": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+			"audit_actions_and_groups": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"storage_account_subscription_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+					v := val.(string)
+					_, err := uuid.FromString(v)
+					if err != nil {
+						errs = append(errs, fmt.Errorf("%q is not in correct format:%+v", key, err))
+					}
+					return
+				},
+			},
+			"is_storage_secondary_key_in_use": {
```

Comment:
this looks like a Computed, Read-Only field - so I think we can remove this?

Reply:
This feature is available on portal. So I have reserved this feature. 
![image](https://user-images.githubusercontent.com/56525716/70106675-3e4b4780-167f-11ea-9346-c3557d53ef81.png)